jlm/hls/backend/rvsdg2rhls/mem-conv.cpp

```diff
@@ -464,6 +470,7 @@ IsDecoupledFunctionPointer(
   visited.insert(output);
 
   bool isDecoupled = false;
+  // Iterate through all users of the output
```

Comment:
That comment adds nothing to the code line below, no? The code line below says exactly the same.

Reply:
With my skills, it's not obvious that dereferencing an output gives you it's users. So this comment helps a user like me.
I've therefore kept it, but if you want I can remove it.